src/daria/corrections/shape/curvature.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, config: Optional[dict] = None, **kwargs) -> None:
                 self.reference_image = im_source
 
             elif isinstance(im_source, str):
-                self.reference_image = np.array(PIL_Image.open(Path(im_source)))
+                self.reference_image = cv2.imread(im_source, cv2.IMREAD_UNCHANGED)
```

Comment:
Shall we also convert to RGB here to not introduce any visual changes to the jupyter notebook on curvature correction?